packages/server/lib/utils/utils.ts

```diff
@@ -10,6 +10,7 @@ import { OrchestratorClient } from '@nangohq/nango-orchestrator';
 
 const logger = getLogger('Server.Utils');
 
+/** @deprecated TODO delete this */
```

Comment:
Would it make sense to just knock this out? It's only used in a couple of spots right now.

Reply:
Yes but I wasn't sure about the fix, just noted for later